llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -2205,7 +2205,26 @@ void SelectionDAGLegalize::ExpandFPLibCall(SDNode* Node,
     Results.push_back(Tmp.first);
     Results.push_back(Tmp.second);
   } else {
-    SDValue Tmp = ExpandLibCall(LC, Node, false).first;
+    bool isSignedArgument;
+    switch (LC) {
```

Comment:
I would hope we would have all of these builtins declared with the signature you would read from, but that seems to not exist

Reply:
Indeed, I've been told in issue #108904 the signatures simply aren't available, so this is the best I can do to fix the regression for now.